docker-compose.dev.yml

```diff
@@ -1,96 +1,100 @@
+        depends_on:
+            db:
+                condition: service_healthy
+            prisma:
+                condition: service_completed_successfully
+        links:
+            - db
 
-  db:
-    image: postgres:16.0-alpine
-    ports:
-      - 5432:5432
-    environment:
-      POSTGRES_DB: ${DB_NAME}
-      POSTGRES_USER: ${DB_USERNAME}
-      POSTGRES_PASSWORD: ${DB_PASSWORD}
-    volumes:
-      - devdb:/var/lib/postgresql/data
-    healthcheck:
-      test: ["CMD-SHELL", "sh -c 'pg_isready -U ${DB_USERNAME} -d ${DB_NAME}'"]
-      interval: 0.5s
-      timeout: 0.5s
-      retries: 60
+    db:
+        image: postgres:16.0-alpine
+        ports:
+            - 5432:5432
+        environment:
+            POSTGRES_DB: ${DB_NAME}
+            POSTGRES_USER: ${DB_USERNAME}
+            POSTGRES_PASSWORD: ${DB_PASSWORD}
+        volumes:
+            - devdb:/var/lib/postgresql/data
+        healthcheck:
+            test:
+                [
+                    "CMD-SHELL",
+                    "sh -c 'pg_isready -U ${DB_USERNAME} -d ${DB_NAME}'"
+                ]
+            interval: 0.5s
+            timeout: 0.5s
+            retries: 60
 
-  prisma:
-    build:
-      context: ./src/prisma
-      dockerfile: ./prismaservice/Dockerfile
-      target: dev
-      args:
-        SHOULD_MIGRATE: ${VEVEN_DB_URI}
-    environment:
-      NODE_ENV: development
-      DB_URI: ${DB_URI}
-      MIGRATE_FROM_VEVEN: ${MIGRATE_FROM_VEVEN}
-      VEVEN_STORE_URL: ${VEVEN_STORE_URL}
-      VEVEN_DB_URI: ${VEVEN_DB_URI}
-      MIGRATION_WITH_LIMITS: ${MIGRATION_WITH_LIMITS}
-    volumes:
-      - ./src/prisma/schema.prisma:/usr/src/app/schema.prisma
-      - devstore:/usr/src/app/prismaservice/store
-    depends_on:
-      db:
-        condition: service_healthy
-    links:
-      - db
+    prisma:
+        build:
+            context: ./src/prisma
+            dockerfile: ./prismaservice/Dockerfile
+            target: dev
+            args:
+                SHOULD_MIGRATE: ${VEVEN_DB_URI}
+        environment:
+            NODE_ENV: development
+            DB_URI: ${DB_URI}
+            MIGRATE_FROM_VEVEN: ${MIGRATE_FROM_VEVEN}
+            VEVEN_STORE_URL: ${VEVEN_STORE_URL}
+            VEVEN_DB_URI: ${VEVEN_DB_URI}
+            MIGRATION_WITH_LIMITS: ${MIGRATION_WITH_LIMITS}
+        volumes:
+            - ./src/prisma/schema.prisma:/usr/src/app/schema.prisma
+            - devstore:/usr/src/app/prismaservice/store
+        depends_on:
+            db:
+                condition: service_healthy
+        links:
+            - db
 
-  nginx:
-    image: nginx:1.21.3-alpine
-    ports:
-      - 80:80
-    volumes:
-      - ./nginx.conf:/etc/nginx/nginx.conf
-      - devstore:/usr/store
-    depends_on:
-      - projectnext
-    links:
-      - projectnext
+    nginx:
+        image: nginx:1.21.3-alpine
+        ports:
+            - 80:80
+        volumes:
+            - nginx/nginx.dev.conf:/etc/nginx/nginx.conf
```

Comment:
Just noticed a './' at the beginning s missing here. This makes docker think that this is a volume instead of a bind. Whoopsee...